.github/workflows/build.yml

```diff
@@ -40,5 +40,8 @@ jobs:
       - name: Verify Plugin
         run: ./gradlew verifyPlugin
 
+      - name: Run Plugin Verifier
+        run: ./gradlew runPluginVerifier
+
```

Comment:
I agree that this task should be the part of "build" workflow. Let's move it into separate job (atm we have `gradleValidation` and `test`) so we can control PR checks and see it separately.

Reply:
@pavel-snyk could you please review last changes?